PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamCorrHists.h

```diff
@@ -264,6 +264,14 @@ class AliFemtoDreamCorrHists {
     fSameEventDistNonCommon[i]->Fill(RelK);
   }
   ;
+    void FillSameEventMultDistCommon(int i, int iMult, float RelK) {
+    if (fSameEventMultDistCommon[i])
+      fSameEventMultDistCommon[i]->Fill(RelK, iMult);
+  }
+    void FillSameEventMultDistNonCommon(int i, int iMult, float RelK) {
+    if (fSameEventMultDistNonCommon[i])
+      fSameEventMultDistNonCommon[i]->Fill(RelK, iMult);
+  }
```

Comment:
Check for !minimalbooking (also above in the fill same event dist non common)

Reply:
done